src/lib.rs

```diff
@@ -141,6 +141,17 @@ where
         /// Amount of frames recommended to be skipped in order to let other clients catch up.
         skip_frames: u32,
     },
+    /// Sent whenever GGRS locally detected a discrepancy between local and remote checksums
+    DesyncDetected{
```

Comment:
I think ggrs uses regular rust formatting, so just run rustfmt to get the recommended style. (Missing a space before the brace here)